app/src/main/java/com/parishod/watomatic/fragment/SettingsFragment.java

```diff
@@ -0,0 +1,39 @@
+package com.parishod.watomatic.fragment;
```

Comment:
@spuday90 We might need a separate activity for Settings.

New settings activity or new settings activity+fragment is what we need.

Or how about this:
Android Studio: `File` ➔ `New` ➔ `Activity` ➔ `Settings Activity`
And trust Android studio will recommend the best.

Reply:
I need to do some research to validate/refute above suggestion but this intuitively feels wrong.

Yes, the entire app can have a single activity and a hundred fragments but what my intuitive(without much reading) suggestion is:
1. Use separate activity wherever you can. Typically every screen (in general sense) can be an activity.
2. Split Activity into fragments wherever it makes sense.
3. (if anything better exists in Android architecture components, go for it)

In our case, according to rule 1, settings can be put in a separate activity so we put it in a separate activity.